src/EnergyPlus/DataDaylightingDevices.hh

```diff
@@ -58,7 +58,7 @@
 
 namespace EnergyPlus {
 
-namespace DataDaylightingDevices {
+namespace Dayltg {
```

Comment:
There are now 2 namespaces with the same name. DataDaylighting.hh and here. I guess that's OK but it seems these should be self identified. Daylighting and DaylightingDevices?

Reply:
They are not two namespaces with the same name, they are just one namespace. Namespaces can be split among multiple files.

Reply:
Another missed lecture.